uint8 control
uint8 gc_l
uint8 gc_r
float64 aea
float64 pea
float64 lhip_pos
float64 rhip_pos
float64 pst
float64 pfl1
float64 pfl2
float64 dst
float64 dfl1
float64 dfl2
float64 lfl
float64 lot
float64 lst
float64 tsw
float64 loc
float64 aover
float64 rco
float64 thip
float64 tab
